### PR TITLE
cluster-tag-filter

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.1 Feature** 

Doc-433

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1059#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 6d7d08d7ceb109c007c046659b40941497e6ab9a | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1059/librarium_build/1605656745827373056) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:33]: Line 30 is incomplete.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:68]:
```suggestion
* Log in to Palette as a Tenant Admin.
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:84]: Line 81: A message displays that confirms the user was created successfully.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:188]: Line 188: Log in to Palette as Tenant Admin.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:30]:
```suggestion
You need to create a filter in Palette:
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:81]:
```suggestion
* A message displays that confirms the user was created successfully.
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:188]:
```suggestion
* Log in to Palette as Tenant Admin
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:226]:
```suggestion
* Select required permissions and save your changes.
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:229]:
```suggestion
* A message displays that confirms the new custom resource role was created successfully.
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:232]:
```suggestion
* From the left main menu, click the **Resource Roles** tab to validate the new role is listed.
```

Comment (ritawatson) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md:232]: It's unclear if the **Resource Roles** tab and **Roles** tab are the same thing. Offered suggestion above.

**Review by ritawatson — APPROVED** [review]
Some minor comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1059#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1059#pullrequestreview-1226876698" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Resma and I spoke about this. The information will be broken up into a few pages to improve readability and make the information more consumable.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md:55]:
```suggestion
##  Paltform Permissions
```
These are platform permissions, not custom roles.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md:63]: Resma, each one of these should be its own table with the following column (Action | Description). The user needs to understand what each permission does. It might seem intuitive but we need to define the action.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md:59]:
```suggestion
The following is a list of permissions and operations supported by Palette. Use these permissions when creating a custom role to control its access.
```

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md:32]:
```suggestion
Palette provides the following global resource roles out-of-the-box:
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions provided.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1059#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1059#pullrequestreview-1226876698" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md
Comment (resmaa) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/01-global-cutome-rroles.md:115]: This is the sub section which tells how to create custom roles.. so i placed it here.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/02-new-user.md
Comment (resmaa) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter/02-new-user.md:1]: yes, as per our conversation this will be moved to user management after this release

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:57]:
```suggestion
Use the following steps to create a custom resource role:
```

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:76]: Guide the user, use the numbered steps approach.

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:79]: This is a how-to as well, so it needs the remaining sections.

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:3]:
```suggestion
metaTitle: "Global and Custom Resource Roles"
```

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:2]:
```suggestion
title: "Global and Custom Resource Roles"
```

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:4]:
```suggestion
metaDescription: "Palette contains global resource roles and supports the ability to create custom resource roles. "
```

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:302]:
```suggestion
[Resource Scope Matrix](/user-management/palette-rbac#resourcescopematrix)
```
resma, I shouldn't have to fix simple issues like this. Slow down and review your changes.

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:19]:
```suggestion
Palette support two types of resource roles, global resource roles and custom resource roles:
```
Use simple terms.

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:23]:
```suggestion
* [Palette Global Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#globalresourceroles) are a set of roles built in and available to you.
```

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:26]:
```suggestion
* [Palette Custom Resource Roles](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#customroles), are roles you can create in Palette using a set of permissions and operations.
```

File: content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/2-palette-rbac/3-resource-scope-roles-permissions.md:27]:
```suggestion

To learn how to create a custom role. Review the [Create Custom Role](/clusters/cluster-management/cluster-tag-filter/global-cutome-rroles#createcustomroles) guide.
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions.

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:52]: Same here,  pretend the user asked "how do I get to the User and Team's page".

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:78]: Same feedback as above.

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:82]:
```suggestion
* Click the **Roles** tab from the left **Main Menu** and click on the **Resource Roles** tab to find the new role name listed.
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback provided below.

File: content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md
Comment (resmaa) [reply content/docs/04-clusters/06-cluster-management/15-cluster-tag-filter.md]: name changed

File: content/docs/04-clusters/06-cluster-management/16-cluster-tag-filter/00-create-add-filter.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/16-cluster-tag-filter/00-create-add-filter.md:94]:
```suggestion
1. Log in to Palette as Tenant Admin
```

File: content/docs/08-user-management/1.5-new-user.md
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1.5-new-user.md:84]:
```suggestion
* The validation can also be done from the **Tenant console**. Go to **Tenant Settings** from the left **Main Menu**.
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1059#pullrequestreview-1230175212" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1059#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1059#pullrequestreview-1226876698" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ritawatson — COMMENTED** [review]
I made a few wording suggestions. I couldn't actually make them 'suggestions', so they are comments instead that you can copy/paste if you like them.

**Pull request merged**
